### PR TITLE
Translate /learn/foundations/how-nextjs-works/index

### PR DESCRIPTION
**Pull request by lucy70381**
Translate /learn/foundations/how-nextjs-works/index

### CODE REVIEW

File: pages/learn/foundations/how-nextjs-works/index.mdx [hunk 1]
```diff
@@ -16,5 +14,17 @@ import NotTranslateYet from '@components/NotTranslateYet';
   ]}
 />
 
-<NotTranslateYet/>
+在您學習更進階的 Next.js 功能前，了解 Next.js 運作的基礎原理會很有幫助。
+
+在本課程的開始，我們討論了對於您如何建置和構建您的應用程式 React 是如何相對地自由 - 有多種方式可使用 React 來建置應用程式。Next.js 提供了一個框架來構建您的應用，可有助於加速開發過程和最終應用程式的優化。
```
Comment (yuaanlin) [inline pages/learn/foundations/how-nextjs-works/index.mdx:19]: 這句話翻譯的不太通順，建議可以先理解原文的意思以後用自己的話重新說一遍，不需要讓每個字和順序都和原文一模一樣沒關係！

**Review by yuaanlin — CHANGES_REQUESTED** [review]
Thanks for the contribution! There is one sentence can be translated more natively. Please help us to improve it!

謝謝你的貢獻！有一句話可以翻譯的更佳流暢，請幫助我們優化它！

File: pages/learn/foundations/how-nextjs-works/index.mdx (hunk 1 above)
Comment (lucy70381) [reply pages/learn/foundations/how-nextjs-works/index.mdx]: The code has been updated, please check.

**Review by yuaanlin — APPROVED** [review]
Nice work! Thanks for the contribution.

**Comment by yuaanlin** [comment]
https://nextjs.tw/learn/foundations/how-nextjs-works

**Pull request merged**